src/commands/sqlite.rs

```diff
@@ -54,8 +54,16 @@ pub struct DeleteCommand {
 #[derive(Parser, Debug)]
 pub struct ExecuteCommand {
     /// Name of database to execute against
-    #[clap(value_parser = clap::builder::ValueParser::new(disallow_empty))]
-    name: String,
+    #[clap(name = "DATABASE", short = 'd', long = "database", value_parser = clap::builder::ValueParser::new(disallow_empty), group = "db", required_unless_present = "LABEL")]
+    database: Option<String>,
+
+    /// Label of database to execute against
+    #[clap(name = "LABEL", long = "label", value_parser = clap::builder::ValueParser::new(disallow_empty), group = "db", requires = "APP", required_unless_present = "DATABASE")]
```

Comment:
Should we add a short option here?

```suggestion
    #[clap(name = "LABEL", long = "label", short = 'l', value_parser = clap::builder::ValueParser::new(disallow_empty), group = "db", requires = "APP", required_unless_present = "DATABASE")]
```

Reply:
Yeah, I guess also allow `-a` for app for consistency with `spin cloud sqlite list`?  I don't think `-a` is used for anything else on the `sqlite` commands.